src/main/routes/port-forward.ts

```diff
@@ -72,20 +72,20 @@ class PortForward {
 
 class PortForwardRoute extends LensApi {
 
-  public async routeServicePortForward(request: LensApiRequest) {
+  public async routePortForward(request: LensApiRequest) {
     const { params, response, cluster} = request
 
     let portForward = PortForward.getPortforward({
-      clusterId: cluster.id, kind: "service", name: params.service,
+      clusterId: cluster.id, kind: params.resourceType, name: params.resourceName,
       namespace: params.namespace, port: params.port
     })
     if (!portForward) {
-      logger.info(`Creating a new port-forward ${params.namespace}/${params.service}:${params.port}`)
+      logger.info(`Creating a new port-forward ${params.namespace}/${params.resourceType}/${params.resourceName}:${params.port}`)
```

Comment:
Please use object-destruct to simplify readability:
`const {resourceType, resourceName, ...etc} = params;`